### ISSUE TITLE
Unable to import into a remix spa mode (vite) app

### ISSUE THREAD

**Issue by neilhsmith** [rocketclimb/rocketicons#132]
Directory import '...\node_modules\rocketicons\core' is not supported resolving ES modules imported from ...\node_modules\rocketicons\lu\index.mjs

I'm in a Remix v2.9.1 spa mode app (vite version is 5.1.0) but it errors with the above as long as any rocketicon import is in the project. I've tried `import { LuLoader2 } from "rocketicons/lu";` and `import { DiAngularSimple } from "rocketicons/di";` but get the same error.

**Comment by amorimjj** [comment]
Hi @neilhsmith, thanks for your report.

I was able to reproduce the issue here. I've taken a look, but I'm not sure how we can fix that. The issue is because we can't force the index.mjs on import because we need React Native to select index.native.mjs when needed.

That said, I was able to run Rocketicons on Remix using --experimental-specifier-resolution=node.

I did that updating the package.json

```
"scripts": {
    ...
    "dev": "NODE_OPTIONS='--experimental-specifier-resolution=node' remix vite:dev",
    "start": "NODE_OPTIONS='--experimental-specifier-resolution=node' remix-serve ./build/server/index.js",
    ...
  },
```

I hope it helps you for now.
